qiskit_ibm_provider/ibm_backend.py

```diff
@@ -104,6 +116,77 @@ class IBMBackend(Backend):
     :class:`BackendJobLimit<qiskit_ibm_provider.BackendJobLimit>` instance::
 
         job_limit = backend.job_limit()
+
+    Here is list of attributes available on the ``IBMBackend`` class:
+        * name: backend name.
+        * backend_version: backend version in the form X.Y.Z.
+        * num_qubits: number of qubits.
+        * target: A :class:`qiskit.transpiler.Target` object for the backend.
+        * basis_gates: list of basis gates names on the backend.
+        * gates: list of basis gates on the backend.
+        * local: backend is local or remote.
+        * simulator: backend is a simulator.
+        * conditional: backend supports conditional operations.
+        * open_pulse: backend supports open pulse.
+        * memory: backend supports memory.
+        * max_shots: maximum number of shots supported.
+        * coupling_map (list): The coupling map for the device
+        * supported_instructions (List[str]): Instructions supported by the backend.
+        * dynamic_reprate_enabled (bool): whether delay between programs can be set dynamically
```

Comment:
What's the source for these attributes? Can we keep it in sync over time?

Reply:
https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/providers/models/backendconfiguration.py#L253
Yes, these don't change too often.

Reply:
They're defined in an external schema here: https://github.com/Qiskit/ibm-quantum-schemas/blob/main/schemas/backend_configuration_schema.json which also was originally based on: https://arxiv.org/abs/1809.03452

Reply:
Thanks for the clarifications. Up to you, @rathishcholarajan if you want to include a reference to the schema and/or original paper here. For users like me it would have been helpful.